projects/step-frontend/src/lib/modules/timeseries/chart/time-series-chart.component.html

```diff
@@ -1,4 +1,11 @@
-<div class="chart-wrapper" [class.pointer]="settings.showExecutionsLinks" (stepElementResize)="resize()">
+<div
+  class="chart-wrapper"
+  [style.height]="height + 'px'"
+  [style.minHeight]="height + 'px'"
+  [style.maxHeight]="height + 'px'"
```

Comment:
https://angular.io/guide/class-binding#binding-to-multiple-styles

```suggestion
  [style.height.px]="height"
  [style.minHeight.px]="height"
  [style.maxHeight.px]="height"
```

Reply:
cool